appengine/endpoints/README.md

```diff
@@ -0,0 +1,35 @@
+# Google Cloud Endpoints sample for Google App Engine
+
+This sample demonstrates how to use Google Cloud Endpoints on Google App Engine Flexible Environment using Node.js.
+
+## Running locally
+
+Refer to the [appengine/README.md](../README.md) file for instructions on
+running and deploying.
+
+## Send an echo request
+
+Choose your local or production server:
+
+```
+# If you're running locally, you won't need an API key.
+$ export ENDPOINTS_HOST=http://localhost:8080
+
+$ export ENDPOINTS_HOST=https://PROJECT-ID.appspot.com
+$ export ENDPOINTS_KEY=AIza...
+```
+
+Send the request:
+
+```
+$ curl -vv -d '{"message":"foo"}' -H 'Content-Type: application/json' "${ENDPOINTS_HOST}/echo?key=${ENDPOINTS_KEY}"
+```
+
+If you're running locally, you won't need an API key.
+
+## Sending authenticated requests
+
+No Node.js client is written yet, but you can try the Python client found [here][python-client].
```

Comment:
No. Will file issues for TODOs for this sample after this is submitted.

There are a few things:
* Client library
* Add tests
* Add error handling